.editorconfig

```diff
@@ -57,3 +57,7 @@ indent_size = 2
 [*.ui]
 indent_style = space
 indent_size = 1
+
+[{*.obt,*.ovt}]
```

Comment:
While not in use yet `.oha` is also a valid extension for high-contrast adjustment layers.

Reply:
Added